ccd-definition/CaseEvent/User/UserEventsCaseProg-nonprod.json

```diff
@@ -67,6 +67,22 @@
     "ShowEventNotes": "N",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "TRIAL_READINESS",
+    "Name": "Trial Readiness",
+    "Description": "Trial Readiness",
+    "PreConditionState(s)": "PREPARE_FOR_HEARING_CONDUCT_HEARING",
+    "PostConditionState": "PREPARE_FOR_HEARING_CONDUCT_HEARING",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/submitted",
+    "ShowSummary": "N",
+    "ShowEventNotes": "N",
+    "RetriesTimeoutAboutToStartEvent": 0,
```

Comment:
I believe this integrates with Camunda ? If so we will need publish : Y

Reply:
I'm not sure about this. I've done the code the way Max advised me, but I will consult him again on this.

Reply:
If this integrates with Camunda (speak to Doug) then you need publish : Y

Reply:
I am not aware as of yet of this integrating with camunda.
There are trial-ready notifications and schedulers in camunda, but they do not relate directly to this event - they are just named similarly.

It seems to me to be fine as is - when the designs for Trial Readiness come back, we should be able to ascertain what needs to go into the event, or if anything needs to integrate with camunda.

Reply:
I talked to Doug, but he said that he doesn't know about this, so he suggested to follow what Max said.